static/app/views/starfish/modules/databaseModule/databaseTableView.tsx

```diff
@@ -163,6 +131,44 @@ export default function DatabaseTableView({
     sortHeader: TableColumnHeader | undefined;
   }>({direction: undefined, sortHeader: undefined});
   const theme = useTheme();
+  const COLUMN_ORDER: TableColumnHeader[] = [
+    {
+      key: 'description',
+      name: 'Query',
+      width: 550,
+    },
+    {
+      key: 'domain',
+      name: 'Table',
+      width: 175,
+    },
+    {
+      key: 'epm',
+      name: 'Throughput (SPM)',
+      width: 175,
+    },
+    {
+      key: 'p50',
+      name: 'P50',
+      width: 175,
+    },
+    {
+      key: 'p95',
+      name: 'P95',
+      width: !p95asNumber ? 175 : undefined,
+    },
+    {
+      key: 'transactions',
+      name: 'transactions',
+    },
+    {
+      key: 'total_time',
+      name: 'Total Time',
+    },
+  ];
+  if (noP95) {
+    COLUMN_ORDER.splice(4, 1);
+  }
```

Comment:
Can ignore this, just an experiment! But I think a filter would have been cleaner here
```suggestion
  if (noP95) {
    COLUMN_ORDER = COLUMN_ORDER.filter(col => col.key !== 'p95')
  }
```

Reply:
Forgot to mention, you'll also need to change it to `let COLUMN_ORDER =` instead of const